support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/config/RegistryConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package org.apereo.cas.config;
+
+import org.apereo.cas.configuration.CasConfigurationProperties;
+import org.apereo.cas.dao.CassandraDao;
+import org.apereo.cas.dao.NoSqlTicketRegistry;
+import org.apereo.cas.dao.NoSqlTicketRegistryDao;
+import org.apereo.cas.logout.LogoutManager;
+import org.apereo.cas.serializer.JacksonJSONSerializer;
+import org.apereo.cas.ticket.registry.TicketRegistry;
+import org.apereo.cas.ticket.registry.TicketRegistryCleaner;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+@Configuration("ticketRegistryConfiguration")
+@EnableScheduling
+@EnableConfigurationProperties({CasConfigurationProperties.class, CassandraProperties.class})
+public class RegistryConfiguration {
+
+    @Autowired private CassandraProperties cassandraProperties;
+
+    @Bean(name = "cassandraDao")
+    public NoSqlTicketRegistryDao cassandraJSONDao() {
+        return new CassandraDao<>(cassandraProperties.getContactPoints(), cassandraProperties.getUsername(), cassandraProperties.getPassword(), new JacksonJSONSerializer(), String.class);
+    }
+
+    @Bean(name = {"noSqlTicketRegistry", "ticketRegistry"})
+    public TicketRegistry noSqlTicketRegistry(final NoSqlTicketRegistryDao cassandraDao, @Qualifier("logoutManager") final LogoutManager logoutManager) {
```

Comment:
Rename the bean and the class. This is not a nosql-ticket registry as it is generically defined here. This is about Cassandra. (There are other nosql registries in CAS).